### ISSUE TITLE
Ready button not doing anything

### ISSUE THREAD

**Issue by TLouison** [TLouison/StoryHop#9]
Not sure if I'm understanding the purpose of the ready button or if it has been implemented yet.  When I click it, it says ready but I can still add to the story regardless. If it hasn't been implemented, some ideas for what to do with it are in the wiki under the [Future Roadmap](https://github.com/TLouison/StoryHop/wiki/Future-Roadmap) page. If there are already plans for it please put them below :)

**Comment by hertacalvo** [comment]
Indeed, it has not been implemented yet, but added as a placeholder for future implementation @TLouison.
